winterfell/webapp/src/templates/form.jade

```diff
@@ -1,4 +1,4 @@
-div.form-wrapper
+div#vfi-form-view
```

Comment:
I was using a class `form-wrapper` and was getting unexpected properties. To avoid this, I could either find another class name or make this class an id instead. To be consistent with the rest of the app, I think this would be best with an id.